profile/profile.go

```diff
@@ -674,6 +674,39 @@ func numLabelsToString(numLabels map[string][]int64, numUnits map[string][]strin
 	return strings.Join(ls, " ")
 }
 
+// SetLabel sets the specified key to the specified value for all samples in the
+// profile.
+func (p *Profile) SetLabel(key string, value []string) {
+	for _, sample := range p.Sample {
+		if sample.Label == nil {
+			sample.Label = map[string][]string{}
+		}
+		sample.Label[key] = value
+	}
+}
+
+// RemoveLabel removes all labels associated with the specified key for all
+// samples in the profile.
+func (p *Profile) RemoveLabel(key string) {
+	for _, sample := range p.Sample {
+		if _, ok := sample.Label[key]; ok {
```

Comment:
https://blog.golang.org/go-maps-in-action: "The delete function doesn't return anything, and will do nothing if the specified key doesn't exist.", so I think you could drop the check?

Reply:
Done